content/en/docs/instrumentation/php/automatic.md

```diff
@@ -57,6 +57,14 @@ function ends it.
       how to do that using a specific version of the extension (1.0.0beta2 in
       this case):
 
+      **Installing from source requires proper development environment and few
+      dependendencies:**
+
+      {{< ot-tabs "Linux (apt)" "MacOS (homebrew)">}}
+      {{< ot-tab lang="shell">}} $ sudo apt-get install gcc make autoconf {{< /ot-tab >}}
+      {{< ot-tab lang="shell">}} $ brew install gcc make autoconf {{< /ot-tab >}}
```

Comment:
```suggestion
      {{< ot-tab lang="shell">}} sudo apt-get install gcc make autoconf {{< /ot-tab >}}
      {{< ot-tab lang="shell">}} brew install gcc make autoconf {{< /ot-tab >}}
```

Reply:
@chalin same as above

Reply:
Right. Not all our files are consistent.

The convention we've being using moving forward is that if a command has no output, we use `lang="shell"` (or `sh`) as you have above, but then omit the prompt `$`.  Any objections to such a change?

Reply:
no, just observation